packages/nodejs/src/instrument.ts

```diff
@@ -53,55 +51,45 @@ export class Instrumentation {
           `Unable to instrument module ${name}, module version needs to satisfy version range ${plugin.version}`
         )
 
-        // abandon the plugin
-        plugin = undefined
-
         return mod
       }
     })
 
-    if (plugin !== undefined) {
-      this.active.push({ name, plugin, hook })
-    }
+    this.active.push({ name, hook })
 
-    return this.active
+    return
   }
 
   /**
    * Removes all custom instrumentation for a given module name. Any
    * subsequent calls to `require` for this instrumentation after calling
    * this method will not include instrumentation.
    */
-  public unload(name: string): InstrumentedModule<any>[] {
+  public unload(name: string): void {
     this.active = this.active.filter(active => {
       if (active.name !== name) {
         return true
       } else {
-        const { plugin, hook } = active
-
-        plugin.uninstall()
```

Comment:
Removed this call as `hook.unhook()` removes the instrumentation altogether from the `require()` cache, making unpatching it unnecessary.